### PR TITLE
Added important workflow tips for VSCode. 

### PR DESCRIPTION
**Pull request by imskr**
Fixed issue #779

### CODE REVIEW

File: vs-code/vscode.md [hunk 1]
```diff
@@ -48,6 +49,28 @@ Most software is written using text editors. Visual Studio Code (VS Code) is a v
...
+
+- **Auto Indent** - VS Code package to format your JavaScript / TypeScript / CSS using [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode).
+
+- **Indent multiple lines** - Just highlight the lines you want, then press tab to indent the entire code block, or **shift+tab** to un-indent the entire code block.
+
+- **Code Folding** - Folds the innermost uncollapsed region at the cursor.<br>
+                     **"Ctrl + Shift + ["** on Windows and Linux.<br>
+                     **"⌥ + ⌘ + ["** on macOS.
+                     
+- **Autosave** - The easiest way to turn on **Auto Save** is with the **File > Auto Save** toggle that turns on and off save after a                      delay.
```
Comment (manaswinidas) [inline vs-code/vscode.md]: This sentence seems incomplete, is there a missing conjunction?

File: vs-code/vscode.md [hunk 2]
```diff
@@ -48,6 +49,28 @@ Most software is written using text editors. Visual Studio Code (VS Code) is a v
 
 [Slides: Local Development with VS Code](https://docs.google.com/presentation/d/1QAMxrS1ZOvtSn7MuYxTmeanrMQi_O8Z5UVq-4LzLqso/edit?usp=sharing)
 
+### VSCode Tips
+
+- **Auto Indent** - VS Code package to format your JavaScript / TypeScript / CSS using [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode).
```
Comment (manaswinidas) [inline vs-code/vscode.md]: I think there is an auto indent option on the menu and it is handy as well. Might be good if you add those instructions as well

File: vs-code/vscode.md (hunk 1 above)
Comment (imskr) [reply vs-code/vscode.md]: Done.

File: vs-code/vscode.md (hunk 2 above)
Comment (imskr) [reply vs-code/vscode.md]: Thank you for pointing out. Added.

File: vs-code/vscode.md [hunk 3]
```diff
@@ -48,6 +49,28 @@ Most software is written using text editors. Visual Studio Code (VS Code) is a v
...
+
+- **Indent multiple lines** - Just highlight the lines you want, then press tab to indent the entire code block, or **shift+tab** to un-indent the entire code block.
+
+- **Code Folding** - Folds the innermost uncollapsed region at the cursor.<br>
+                     **"Ctrl + Shift + ["** on Windows and Linux.<br>
+                     **"⌥ + ⌘ + ["** on macOS.
+                     
+- **Autosave** - The easiest way to turn on **Auto Save** is with the menu **File > Auto Save** toggle that turns on and off save after                  a delay.                    
+
+- **Key-Value** - Visual Studio Code lets you perform most tasks directly from the keyboard.
```
Comment (alodahl) [inline vs-code/vscode.md]: I don’t think titling this section “key-value” helps learners understand what those terms mean. Choose a title more about how this fits into vscode, and add a section on what a key-value pair is. Apprentices will do this in week 1, about a month before they learn about JS objects.

Comment (imskr) [reply vs-code/vscode.md]: ok

File: vs-code/vscode.md [hunk 4]
```diff
@@ -48,6 +49,28 @@ Most software is written using text editors. Visual Studio Code (VS Code) is a v
...
+                  Each rule consists of:
+     - A **key** that describes the pressed keys.
+     - A **command** containing the identifier of the command to execute.
+     - An optional **when** clause containing a boolean expression that will be evaluated depending on the current context.
+     
+     **Example**: Keybindings that are active when the focus is in the editor.<br>
+          { "key": "home",            "command": "cursorHome",                  "when": "editorTextFocus" },<br>
```
[the remark this answers is not in the document]
Comment (imskr) [reply vs-code/vscode.md]: let me check again. I will fix this.

**Comment by imskr** [comment]
@alodahl Please check, Is it good now?

**Comment by imskr** [comment]
I'm closing this as i accidentally deleted the branch. I will reopen it again. @alodahl

**Pull request closed without merging**